src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/acquireDotnetInstall.ts

```diff
@@ -0,0 +1,29 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+
+interface IDotnetAcquireResult {
+    dotnetPath: string;
+}
+
+export async function acquireDotnetInstall(outputChannel: vscode.OutputChannel): Promise<string> {
+    const version = "5.0.0";
+    const requestingExtensionId = "ms-blazorwasm-companion";
+
+    try {
+        const dotnetResult = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet.acquire', { version, requestingExtensionId });
```

Comment:
/cc @sfoslund Any recommendations here on API usage that we should be aware of? Especially in regards to how we're capturing error output

One piece I can think of is depending on what the debug proxy does under the cover you'll probably need to utilize the [`ensureDotnetDependencies`](https://github.com/dotnet/vscode-dotnet-runtime/blob/bd166d4770c682394efd578f7cfc9cf221d3b7a7/sample/src/extension.ts#L55) to enable things don't explode on arch linux environments

Reply:
Looks good for the most part other than my comment on 2 part version numbers. In the event of an error the runtime extension will surface its own error popup with what went wrong, so this error handling should be sufficient.